index.html

```diff
@@ -4,106 +4,374 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>document</title>
     <link rel="stylesheet" href="styles/index.css" />
-    <script src="index.js" defer></script>
+    <script type="module" src="index.js" defer></script>
   </head>
   <body class="body">
-    <header><h1 class="headline">Quiz-App</h1></header>
-    <main class="main">
-      <div class="card-set">
-        <svg
-          class="card-set__bookmarklabel js-bookmark"
-          xmlns="http://www.w3.org/2000/svg"
-          width="24"
-          height="24"
-          viewbox="0 0 24 24"
-        >
-          <path d="M18 24l-6-5.269-6 5.269v-24h12v24z" />
-        </svg>
-
-        <h2 class="bold">Question</h2>
-        <p class="card-set__blocksatz">
-          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
-          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
-          sed diam voluptua?
-        </p>
+    <main data-js="main-home" class="main hidden">
+      <h1 class="headline">Quiz-App</h1>
+      <div class="card-sections">
+        <div data-js="card-set" class="card-set">
+          <svg
+            data-js="bookmark"
+            class="card-set__bookmarklabel"
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewbox="0 0 24 24"
+          >
+            <path d="M18 24l-6-5.269-6 5.269v-24h12v24z" />
+          </svg>
 
-        <button class="button js-btn-answer" type="button">
-          <span class="js-btn-showAnswer">show</span>
-          <span class="js-btn-hideAnswer hidden">hide</span>
-          answer
-        </button>
-        <h2 class="bold js-answer-heading hidden">Answer</h2>
-        <p class="card-set__blocksatz js-answer-text hidden">
-          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
-          nonumy eirmod tempor invidunt ut labore et.
-        </p>
+          <h2 class="bold">Question</h2>
+          <p class="card-set__blocksatz">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua?
+          </p>
+
+          <button data-js="btn-answer" class="button" type="button">
+            <span data-js="showAnswer">show</span>
```

Comment:
I would decide for myself which notation i would use. If u sometims type "showAnswer" and sometimes "show-answer, that could cause some issues later

Reply:
Sorry i just saw that @julialoeschel  already adressed the same thing further down. Maybe were soulmates 🥰